c/tskit/core.c

```diff
@@ -289,15 +289,15 @@ tsk_strerror_internal(int err)
             break;
         case TSK_ERR_MUTATION_TIME_YOUNGER_THAN_NODE:
             ret = "Mutations must have a time equal to or greater than that of their "
-                  "node";
+                  "node or equal to NAN";
```

Comment:
It's pedantic, but nothing is equal to NaN AFAIK. More correct to say "... of their or be a NaN value."